src/session-pool.js

```diff
@@ -109,221 +113,178 @@ function SessionPool(database, options) {
 
   this.request_ = database.request;
   this.requestStream_ = database.requestStream;
-
-  this.reads_ = [];
-  this.writes_ = [];
-  this.borrowed_ = [];
-
-  this.pendingCreates_ = 0;
-  this.acquireQueue_ = new PQueue({concurrency: 1});
   this.requestQueue_ = new PQueue({concurrency: this.options.concurrency});
-
-  this.evictHandle_ = null;
-  this.pingHandle_ = null;
-
+  this.acquireQueue_ = new PQueue({concurrency: 1});
```

Comment:
I think we're missing a test for this.

Reply:
This is implicitly tested by other tests cases.